dev-scheme/guile/guile1-1.8.8.recipe

```diff
@@ -0,0 +1,146 @@
+SUMMARY="GNU Ubiquitous Intelligent Language for Extensions"
+DESCRIPTION="Guile is a library designed to help programmers create flexible \
+applications. Using Guile in an application allows programmers to write \
+plug-ins, or modules (there are many names, but the concept is essentially \
+the same) and users to use them to have an application fit their needs."
+HOMEPAGE="https://www.gnu.org/software/guile/"
+COPYRIGHT="1993-2018 Aubrey Jaffer, George Carrette, et al."
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://ftpmirror.gnu.org/guile/guile-$portVersion.tar.gz
+	https://ftp.gnu.org/gnu/guile/guile-$portVersion.tar.gz"
+CHECKSUM_SHA256="c3471fed2e72e5b04ad133bbaaf16369e8360283679bcf19800bc1b381024050"
+PATCHES="guile1-$portVersion.patchset"
+SOURCE_DIR="guile-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libguileVersion="17.4.0"
+libguileVers="${libguileVersion%.*}"
+libguileVersionCompat="$libguileVersion compat >= ${libguileVersion%%.*}"
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
+portVers="${portVersion%.*}"
+
+PROVIDES="
```

Comment:
It would be better to split up the main packe. It should only contain the runtime libs, nothing else, and the commands should go into guile_tools or something, so the 2 different runtime could be installed parallel.

Reply:
You should change the 2.x recipe too to accomplish this.

Reply:
It seems a viable strategy, better than my hack. Maybe the libraries should go into guile1_lib and the executable into a guile1 package. What do you think? Where I can read about splitting a package?

Reply:
OK. I found the docs. I'm implementing the change with libraries in guile1 and tools in guile1_tools as you suggest (seems the most obvious way, given the docs)

Reply:
> You should change the 2.x recipe too to accomplish this.

There is no need now to split also the guile 2.x recipe. There is no runtime conflict anymore between libraries and is natural that the more recent package provide both libraries and executable. It will be a unnecessary complexification to split all the packages without reason. I would tend to leave the guile 2.x recipe as it is now. Everything works at present.